SPECS/samba.spec

```diff
@@ -41,7 +40,7 @@
 
 # XCP-ng: don't build vfs_glusterfs
 %global with_vfs_glusterfs 0
-
+%global with_vfs_cephfs 0
 %global with_intel_aes_accel 0
```

Comment:
Upstream package sets this to 1 in x86_64. What's the impact?

Reply:
It should be mentioned somewhere. Now, I know, but others who may come later to the spec file don't.